pkgs/development/python-modules/qiskit-aer/default.nix

```diff
@@ -61,16 +76,10 @@ buildPythonPackage rec {
     pybind11
```

Comment:
Shouldn't this be in `nativeBuildInputs`?

Reply:
Moving it to ``nativeBuildInputs`` causes build failure in downstream ``qiskit`` b/c it expects pybind11 to be installed for qiskit-aer. https://github.com/NixOS/nixpkgs/pull/121606#issuecomment-849592645